contrib/confluence/src/python/pants/contrib/confluence/util/confluence_util.py

```diff
@@ -7,9 +7,13 @@
 import getpass
 import logging
 import mimetypes
-import urllib
+import urllib.parse
+from builtins import object, str
 from os.path import basename
 
+from future import standard_library
+standard_library.install_aliases()
```

Comment:
Hey this won't work as we're expecting, because `standard_libary.install_aliases()` has to be called before `import urllib.parse`, which is impossible with the iSort tool.

Instead, please do this
```python
from future.moves.urllib.parse import quote_plus
```

And then update line 83